backend/app/gzac/src/main/resources/config/application.yml

```diff
@@ -131,20 +143,14 @@ spring:
                                 password: guest
                                 virtual-host: /
     rabbitmq:
-        host: localhost
-        port: 5672
-        username: guest
-        password: guest
+        ssl.enabled: ${SPRING_RABBITMQ_SSL_ENABLED:true}
+        host: ${SPRING_RABBITMQ_HOST}
+        port: ${SPRING_RABBITMQ_PORT}
+        username: ${SPRING_RABBITMQ_USERNAME}
+        password: ${SPRING_RABBITMQ_PASSWORD}
         publisher-confirm-type: correlated
         publisher-returns: true
 
-logging:
-    level:
-        org.springframework.beans: DEBUG
```

Comment:
Where do we set this info now? This logging is very useful for development.

Reply:
Added these properties to the .env.properties.